tensorrt_llm/_torch/models/modeling_radio.py

```
@@ -1012,11 +1118,22 @@ def load_weights(self, weights):
             if not m.startswith('model.blocks.'):
                 raise ValueError(f"Missing key: {m}")
         for u in unexpected_keys:
-            # TODO(TRTLLM-11269): Add support for the conv3d layer.
+            # model.blocks weights are loaded separately below via
+            # _load_weights_impl (to handle qkv splitting).
+            # video_embedder is only unexpected when temporal_patch_size==1
+            # (model doesn't have the submodule).
             if not u.startswith(
                 ('model.blocks.', 'model.patch_generator.video_embedder')):
```

Flake8 (7.3.0)

[error] continuation line with same indent as next logical line

(E125)